src/cms/models/media/media_file.py

```diff
@@ -130,6 +130,21 @@ def url(self):
         """
         return BASE_URL + self.file.url if self.file else None
 
+    @property
+    def thumbnail_url(self):
+        """
+        Return the path of the image that should be used as the thumbnail.
+
+        :return: The path of the file. If the file is an image and no thumbnail could be generated the file itself will be returned.
+        :rtype: str
+        """
+        if not self.thumbnail:
+            if self.type.startswith("image"):
+                #: Returns the path to the file itself
+                return self.url
```

Comment:
```suggestion
                return BASE_URL + self.url
```
For consistency reasons, it's probably good to append the base url here as well...